src/rollup/RollupAdminLogic.sol

```diff
@@ -44,7 +42,8 @@ contract RollupAdminLogic is RollupCore, IRollupAdmin, DoubleLogicUUPSUpgradeabl
             1,
             IGasRefunder(address(0)),
             0,
-            1
+            1,
+            ""
```

Comment:
We can pass an empty quote here because we allow the RollupAdmin to post the batch without the quote

Reply:
Why do we allow the rollupadmin to post the batch without the quote?

Reply:
1. Yes, [rollupadmin](https://github.com/EspressoSystems/nitro-contracts/blob/integration/src/rollup/RollupAdminLogic.sol) is a contract so it cant possibly run in a TEE
2. It produces the genesis batch (code https://github.com/EspressoSystems/nitro-contracts/blob/integration/src/rollup/RollupAdminLogic.sol#L41). Its how Arbitrum wanted to work, I dont know the reasoning why rollupadmin produces it but it does. Also, its not just the genesis block, if you look at the current method to add batch [here](https://github.com/EspressoSystems/nitro-contracts/blob/integration/src/bridge/SequencerInbox.sol#L476), rollupadmin can always post batches.